frontend/src/components/utils/HeaderNetworkDropdown.tsx

```diff
@@ -18,13 +24,13 @@ const HeaderDropdownItem = ({ link, title }) => {
   );
 };
 
-const HeaderNetworkDropdown = () => (
+const HeaderNetworkDropdown: FC = () => (
   <NetworkConsumer>
     {(context) => (
       <Dropdown>
         <Dropdown.Toggle className="header-network" variant="secondary">
           <div className="network-icon"></div>
-          {context.currentNearNetwork.name}
+          {context.currentNearNetwork?.name}
```

Comment:
It seems like this file shouldn't be rendered if context is empty.
Though context should either provide sane fallbacks or props should be passed from the outer component.
Another issue to address later.

Reply:
Good catch